src/main.py

```diff
@@ -48,10 +53,10 @@ def parse_args():
 
     tonation = tonation_recognition.get_tonation(sounds)
 
-    chords = chords_generation.get_chords(sounds, tonation, meter, sounds[0].timestamp)
+    chords = chords_generation.get_chords(sounds, tonation, (4, 8))
 
     print("Chords:")
-    print("\n".join([f"{chord.timestamp:.3f}: {chord}\t{chord.duration_ms:.3f}" for chord in chords]))  # noqa
+    print("\n".join([f"{chord}\t{chord.duration:.3f}" for chord in chords]))  # noqa
```

Comment:
Why duration, and not duration_ms? Is this not in miliseconds?

Reply:
Look at documentation